app/webpacker/components/Panel/pages/AnonymizationScriptPage/index.jsx

```diff
@@ -1,25 +1,56 @@
 import React from 'react';
-import WcaSearch from '../../../SearchWidget/WcaSearch';
+import { FormField, FormGroup, Radio } from 'semantic-ui-react';
+import { IdWcaSearch } from '../../../SearchWidget/WcaSearch';
 import SEARCH_MODELS from '../../../SearchWidget/SearchModel';
 import useInputState from '../../../../lib/hooks/useInputState';
-import AnonymizationTicketWorkbench from '../../../Tickets/TicketWorkbenches/AnonymizationTicketWorkbench';
+import AnonymizationTicketWorkbenchForWrt from '../../../Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt';
+
+const MODEL_NAME = {
+  [SEARCH_MODELS.user]: 'User',
+  [SEARCH_MODELS.person]: 'Person',
+};
```

Comment:
What about using the keys of this object as a separate `AVAILABLE_MODELS` const (name tentative)?
Would be useful in a few places further down